hbp_knowledge/tfeb/sardiello2016.bel

```diff
@@ -200,138 +236,184 @@ bp(GO:"response to transition metal nanoparticle") -> tloc(p(HGNC:TFEB),fromLoc(
 act(p(HGNCGENEFAMILY:"MTOR complex 1"),ma(kin)) -> p(HGNC:TFEB,pmod(Ph))
 act(p(HGNCGENEFAMILY:"MTOR complex 1"),ma(kin)) -> p(HGNC:TFEB,loc(GO:cytosol))
 p(HGNC:TFEB,pmod(Ph)) -> p(HGNC:TFEB,loc(GO:cytosol))
+UNSET Confidence
 
 SET Evidence = "In cells cultured in nutrient-poor conditions, knockout/down of tuberous sclerosis 2 (Tsc2), an upstream regulator of mTORC1, resulted in partial dephosphorylation and nuclear relocation of endogenous TFEB, which was reverted by rapamycin administration and required a serine-rich region at the TFEB C-terminus.[27]"
+SET Confidence = "High"
 #nutrient-poor conditions
 p(HGNC:TSC2) pos p(HGNCGENEFAMILY:"MTOR complex 1")
 p(HGNC:TSC2) -> p(HGNC:TFEB,pmod(Ph))
 p(HGNC:TSC2) -| tloc(p(HGNC:TFEB),fromLoc(GO:cytosol),toLoc(GO:nucleus))
 a(CHEBI:sirolimus) -> composite(a(CHEBI:sirolimus),p(HGNC:TFEB,frag("?_*")))
 composite(a(CHEBI:sirolimus),p(HGNC:TFEB,frag("?_*"))) -> tloc(p(HGNC:TFEB),fromLoc(GO:cytosol),toLoc(GO:nucleus))
+UNSET Confidence
 
 SET Evidence = "In nutrient-replete cells, inhibition of mTORC1 promoted nuclear accumulation of TFEB by abrogation of mTORC1 phosphorylation of TFEB at Ser142 [28] and Ser211.[29, 30]"
 #in nutrient rich cells
+SET Confidence = "High"
 p(HGNCGENEFAMILY:"MTOR complex 1") -> p(HGNC:TFEB,pmod(Ph,Ser,142))
 p(HGNCGENEFAMILY:"MTOR complex 1") -> p(HGNC:TFEB,pmod(Ph,Ser,211))
 p(HGNCGENEFAMILY:"MTOR complex 1") -| tloc(p(HGNC:TFEB),fromLoc(GO:cytosol),toLoc(GO:nucleus))
 p(HGNC:TFEB,pmod(Ph,Ser,142)) -| tloc(p(HGNC:TFEB),fromLoc(GO:cytosol),toLoc(GO:nucleus))
 p(HGNC:TFEB,pmod(Ph,Ser,211)) -| tloc(p(HGNC:TFEB),fromLoc(GO:cytosol),toLoc(GO:nucleus))
+UNSET Confidence
 
 SET Evidence = "Phosphorylation of TFEB at Ser211 promotes TFEB interaction with the YWHA/14-3-3 proteins,[29, 30] which determine TFEB cytosolic retention by masking a nuclear localization signal located between amino acids 241 and 252.[30]"
+SET Confidence = "High"
 p(HGNC:TFEB,pmod(Ph,Ser,211)) -> composite(p(HGNC:TFEB),p(HGNC:YWHAQ))
 composite(p(HGNC:TFEB),p(HGNC:YWHAQ)) -> p(HGNC:TFEB,loc(GO:cytosol))
 p(HGNC:TFEB,frag("241_252")) isA a(MESH:"Nuclear Localization Signals")
 composite(p(HGNC:TFEB),p(HGNC:YWHAQ)) -| tloc(p(HGNC:TFEB),fromLoc(GO:cytosol),toLoc(GO:nucleus))
+UNSET Confidence
 
 SET Evidence = "The interaction between TFEB and mTORC1 occurs at the lysosomal surface and is indeed abolished by dominant-negative mutations in Rag GTPases (Rags),[28] proteins that mediate the recruitment of mTORC1 to the lysosomal membrane."
+#it is abolished due to negative mutation of Rags, I aint sure whether you could code that with complex.
+SET Confidence = "Medium"
 SET MeSHAnatomy = "Lysosomes"
+#complex(GO:"Ragulator complex") -| composite(p(HGNCGENEFAMILY:"MTOR complex 1"),p(HGNC:TFEB))
 complex(GO:"Ragulator complex") -> tloc(p(HGNCGENEFAMILY:"MTOR complex 1"),fromLoc(GO:cytoplasm),toLoc(GO:"plasma membrane"))
 UNSET MeSHAnatomy
+UNSET Confidence
```

Comment:
@trushaadeshara8  look at my comment here.

Reply:
But isn't that why we can use composite instead of complex

Reply:
@trushaadeshara8 okay. But you still have used complex up and not composite.